extension/content.css

```diff
@@ -1,13 +1,11 @@
-.highlighted {
-	color: #017f92;
-}
-
 mark {
-	background-color:#c9daf8 !important;
-	background: #c9daf8 !important;
-	transition: all 0.5s ease 0.5s !important;
 }
-.marked {
-	background: #c9daf8 !important;
+.markedOne {
 	background-color:#c9daf8 !important;
+		/* background: #c9daf8 !important; */
+}
+.markedTwo {
+	background-color:#e6effd !important;
```

Comment:
for my sake, if you can comment on what these colors are "e.g. light blue" that would be helpful

Reply:
* comment in the code

Reply:
Good idea!